modules/installation-and-upgrade/pages/container-deployment/actkey-bootstrap-proxy-suma.adoc

```diff
@@ -17,12 +25,12 @@
 . Click btn:[Bootstrap].
 . Wait for the bootstrap process to complete successfully.
   Check the menu:Salt[] menu and confirm the {salt} key is listed and accepted.
+// FIXME: Can we skip the following two reboots on both Micro *and* SLES?
```

Comment:
We need to verify this from our QE squad. Howerver, I don't believe these 2 and then next on line 36 after updating the proxy Host are needed for sales. So we should rather like it like

`Reboot the proxy host if sle micro ` or something like this.

For now I would say, make the change by adding the if sle micro explicitly in the docs. QE can then test based on those steps and let us know if that's correct or we need any modification.

@nodeg FYI

Reply:
IIRC the reboots were only introduced and necessary for transactional systems like SL Micro and the way they work. When using e.g. SLES as base OS, no reboot is required since changes can be made right away.